src/include/btree_inline.h

```diff
@@ -390,64 +390,60 @@ __wt_cache_page_inmem_incr(WT_SESSION_IMPL *session, WT_PAGE *page, size_t size,
 
     bool is_disagg = __wt_conn_is_disagg(session);
 
-    /*
```

Comment:
I don't think this makes sense any more. This only ensures that btree size cannot be larger than cache size. I don't think that matters even btree size becomes larger than the cache size temporarily.